src/diff/index.js

```diff
@@ -218,6 +231,7 @@ export function diff(
 
 		if ((tmp = options.diffed)) tmp(newVNode);
 	} catch (e) {
+		newVNode._original = null;
```

Comment:
Won't this cause suspended vnodes to all be considered strictly equal? (vs `newVNode._original = newVNode`)

Reply:
No, if we'd do that we'd keep the reference which in case of error could cause erroneous scenario's

Reply:
Is there a test for this specific item?

Reply:
Everything in `lifecycles/componentDidCatch` and `lifecycles/getDerivedStateFromError` would be testament to this.

What happens is

```
const App = () => (
<Boundary>
  <StaticThrowingChild />
<Boundary />
);
```

When the `Boundary` catches the error the child won't have changed since the only createElement that child is subject to is the one from `App`.

Reply:
Absolutely, this makes sense.

However, for authors 6 months from now (including us!) a test that specifically calls out this equality scenario might be prudent.

Wouldn't call it a blocker, more of a nit.

Reply:
Yes, absolutely will do for sure it's a good shout. Will also document that, this is more of a first view since tests are failing for suspense either way.

Reply:
I think this bit will likely be in conflict with the hydration PR, I'll try to get it updated so we can compare. My hope is that the `_original=null` piece ends up being unnecessary as a result.